app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -74,6 +74,12 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
     // The Google Map instance.
     private lateinit var googleMap: GoogleMap
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        buildMap()
+    }
+
```

Comment:
Moved onCreate function to the top of the code. Moved piece of code to a separate function - buildMap(), created a better organization of code structure.(https://kotlinlang.org/docs/coding-conventions.html#source-file-organization)